hcm:

  #IMU
  imu_timeout_duration: 0.1 # time without messages from the IMU till error is produced [s]

  #Pressure sensors
  pressure_timeout_duration: 0.1 # time without messages from the foot pressure sensors till error is produced [s]
  pressure_sensors_installed: false
  foot_zero_service: "set_foot_zero"

  # Motors
  motor_off_time: 30000000 # time of no use or updates when the hcm goes to soft off
  motor_timeout_duration: 0.1 # time without messages from the servos till error is produced [s]

  #Animations
  anim_server_wait_time: 1 # time the hcm waits for animation server
  animations:
    walkready: "walkready"
    walkready_sim: "walkready_sim"
    falling_front: "falling_front"
    falling_back: "falling_back"
    falling_left: "falling_left"
    falling_right: "falling_right"
    stand_up_front: "stand_up_front"
    stand_up_back: "stand_up_back"
    stand_up_left: "turning_back_left"
    stand_up_right: "turning_back_right"
    sit_down: "walkready"
    penalty: "walkready"
    motor_off: "walkready"
    init: "init"
    anim_package: "wolfgang_animations"
    walkready_pose_threshold: 5 # [deg]

  # Falling
  stand_up_active: true # enables the robot to stand up automatically
  falling_active: true # enables the check for falling and coresponding counter meassurements
  #Grenzwerte sch채rfer --> fruehere Reaktion aber haeufigere Fehlinterpretation, zb. beim Laufen
  #Grenzwerte sanfter --> zu spaete Raktion
  threshold_gyro_y_front: 7 # > gyroY
  threshold_gyro_x_side: 7 # > gyroX
  falling_threshold_orientation_left_right: 60 # > Point of no return in degrees
  falling_threshold_orientation_front_back: 45 # < Point of no return in degrees
  falling_new_front: 32
  falling_new_back: -6
  falling_new_left: -18
  falling_new_right: 18

  falling:

    #falling animations
    "falling_front": "falling_front"
    "falling_back": "falling_back"
    "falling_left": "falling_left"
    "falling_right": "falling_right"

    "wolfgang":

      #Grenzwerte m체ssen auf h채rterem boden auch h채rter eingestellt werden,
      #dies erfolgt mit dem boden_koeffizient
      "ground_coefficient": 1.0  #WEICH(kunstrasen Spielfeld)= 1.0 , HART(teppich Spielfeld)= 1.25

      #false = motoren werden bei fallerkennung abgeschaltet
      "dyn_falling_active": true
